src/back/auth/src/repo/table_session.hpp

```diff
@@ -21,9 +21,18 @@ class Session final {
 
 	model::Session GetById(const std::string& id, bool isActive = true);
 
+	model::Session Get(const std::string& id);
+
 	void UpdateTokens(const model::Session& s);
 
-	void MarkInactive(const model::Session& s);
+	void BlockEverySessionByUser(const std::string& userId);
```

Comment:
Session - есть в названии класса, Every понятно из того что нет специфичных аргументов. Тут можно было бы сделать название покороче, например BlockByUserId. Но у нас уже есть функция которая делает тоже самое но с другими параметрами. Нужно назвать её так же MarkInactive. И ещё поставь их пожалуйста рядом.
```suggestion
	void MarkInactive(const std::string& userId);
```

Reply:
Готово.